Debian/8/Makefile

```diff
@@ -36,6 +37,9 @@ ifeq ($(OVAL_5_11), 0)
 	# Update requested debian8 OVAL version in $(CONF)/oval.config to be the 5.11 OVAL language version
 	sed -i "s/^\(debian8_oval_version = \).*/\15.11/" $(CONF)/oval\.config
 endif
+# 	if needed, list found orphans
+#	$(LOCAL_UTILS)/find_orphans.py $(PROD)|sort
+	$(LOCAL_UTILS)/find_orphans.py $(PROD) | xargs rm
```

Comment:
@redhatrises : I'm updating the Makefile il order to keep the XML validation but deactivate the reference check while the xccdf content is not yet fulfilled with shared oval references. I'll push after a global validate check.